easybuild/easyconfigs/b/BEDTools/BEDTools-2.28.0-foss-2018b.eb

```diff
@@ -0,0 +1,39 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.28.0'
```

Comment:
Any particular reason why not to go with 2.29.2?

Reply:
@Micket Yes, this is in preparation for a (yet to do) `MIRACUM-Pipe` easyconfig and they explicitly want this version: https://github.com/AG-Boerries/MIRACUM-Pipe/blob/master/tools/install.sh#L7. I don't know if there is a real reason behind this but for the start I wanted to do it as they suggest.